cdc/owner/changefeed.go

```diff
@@ -632,6 +648,26 @@ func (c *changefeed) asyncExecDDLEvent(ctx cdcContext.Context,
 	return done, nil
 }
 
+func (c *changefeed) filterDDLEvent(ddlEvents []*model.DDLEvent) []*model.DDLEvent {
```

Comment:
I don't know if it's a good idea to iterate over it and copy it again? Why don't we filter it in the loop above?
Maybe it's acceptable, but should we be able to pre-assign lengths?

Reply:
If we iterate over it to filter, it is needed to do this twice since we have redo and ddl sink to sent.